factory/glideFactoryMonitorAggregator.py

```diff
@@ -265,14 +268,22 @@ def aggregateStatus(in_downtime):
         # load entry status file
         status_fname=os.path.join(os.path.join(monitorAggregatorConfig.monitor_dir, 'entry_'+entry),
                                   monitorAggregatorConfig.status_relname)
+        # load entry completed data file
+        completed_data_fname=os.path.join(os.path.join(monitorAggregatorConfig.monitor_dir,'entry_'+entry),
+                                  monitorAggregatorConfig.completed_data_relname)
         try:
             entry_data=xmlParse.xmlfile2dict(status_fname)
+            completed_data_fp=open(completed_data_fname)
+            completed_data=json.load(completed_data_fp)
         except IOError:
             continue # file not found, ignore
 
         # update entry
         status['entries'][entry]={'downtime':entry_data['downtime'], 'frontends':entry_data['frontends']}
 
+        # update completed data
+        completed_data_tot['entries'][entry]=completed_data['stats']
```

Comment:
Is the 'stat' key always present? Otherwise please use something like completed_data.get('stats', 'Unknown')

Reply:
The 'stats' key will always be present. When the completed_data.json file is written, the data is written to the 'stats' key. However, this does rely on no one changing how completed_data.json is written. Should I make your suggested change for added robustness?